mir_eval/util.py

```diff
@@ -138,6 +138,8 @@ def interpolate_intervals(intervals, labels, time_points, fill_value=None):
         The ``i`` th interval spans time ``intervals[i, 0]`` to
         ``intervals[i, 1]``.
 
+        Intervals are assumed to be disjoint.
```

Comment:
Is this what you mean?  You wrote "Rewrite util.interpolate_intervals to support discontinuities".  From that statement it sounds like the problem is that the intervals are assumed to fully cover the track, not that they are assumed to be disjoint.  Maybe I am confused.